lib/goodcity/organisation_populator.rb

```diff
@@ -38,8 +37,9 @@ def run
 
 
     private
-    def get_organisation_type
-      OrganisationType.find_by_name_en(ORGANISATION_TYPE_NAME) || OrganisationType.create(name_en: ORGANISATION_TYPE_NAME)
+
+    def organisation_type
+      @organisation_type ||= OrganisationType.find_or_create_by(name_en: ORGANISATION_TYPE_NAME)
```

Comment:
@trojanh It looks like you were looking up the organisation_type in the initializer in order to only do it once and not repeat for each loop iteration. A useful trick here is to use the `||=` operator to memoize the lookup. The first time the function is called, it will call find_or_create_by, but the subsequent times, it will just return the previous (non-nil) value and is therefore effectively cached. This has the advantage of extracting another DB call out of the initializer.